test/dlc_tests/container_tests/bin/efa/test_nccl_tests_using_efa

```diff
@@ -90,11 +67,15 @@ check_efa_nccl_all_reduce(){
     # the container. Not using full-paths of mpirun and other executables because these paths can change across PyTorch
     # versions in DLC images.
     mpirun -x FI_PROVIDER="efa" -n $NODES -N $GPU_COUNT --hostfile $NUM_HOSTS_file \
-        -x NCCL_DEBUG=INFO ${USE_DEVICE_RDMA_ARG} -x NCCL_PROTO=simple -x NCCL_ALGO=ring -x RDMAV_FORK_SAFE=1 \
```

Comment:
Why remove the `USE_DEVICE_RDMA_ARG` here? This test should still work for p3dn.24xlarge, right?

Reply:
will change back